Lib/ssl.py

```diff
@@ -427,17 +477,6 @@ class SSLContext(_SSLContext):
     sslsocket_class = None  # SSLSocket is assigned later.
     sslobject_class = None  # SSLObject is assigned later.
 
-    def __new__(cls, protocol=None, *args, **kwargs):
```

Comment:
Maybe a topic of discussion here.
I tested your branch and seen that doing

```
from ssl import SSLContext, PROTOCOL_TLS_SERVER

# does not work anymore
ctx = SSLContext(protocol=PROTOCOL_TLS_SERVER)
# must do
ctx = SSLContext(PROTOCOL_TLS_SERVER)
```

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: _SSLContext() takes no keyword arguments
```

Users may be confused after this change. Maybe make _SSLContext capable of receiving keyword args.
WDYT?